content/en-us/reference/engine/libraries/bit32.yaml

```diff
@@ -238,6 +238,21 @@ functions:
         summary: ''
     tags:
     code_samples:
+  - name: bit32.byteswap
+    summary: |
+      Returns the given number with swapped endianness.
+    description: |
+      Returns the given number with swapped endianness.
```

Comment:
```suggestion
      Returns the given number with the order of the bytes swapped.
    description: |
      Returns the given number with the order of the bytes swapped.
```

Reply:
Suggesting change to "Returns the given number with the order of the bytes swapped." to match the Roblox Luau docs: https://luau.org/library#bit32-library